iOSNativeTemplate/iOSNativeTemplate/AppDelegate.m

```diff
@@ -77,19 +77,24 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
     self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
     [self initializeAppViewState];
-    
+
     // If you wish to register for push notifications, uncomment the line below.  Note that,
     // if you want to receive push notifications from Salesforce, you will also need to
     // implement the application:didRegisterForRemoteNotificationsWithDeviceToken: method (below).
 //    [self registerForRemotePushNotifications];
-    
+
     //Uncomment the code below to see how you can customize the color, textcolor, font and fontsize of the navigation bar
 //    [self customizeLoginView];
-    
+
     [SFSDKAuthHelper loginIfRequired:^{
         [self setupRootViewController];
     }];
-    return YES;
+
+    self.moduleName = @"RnDiffApp";
```

Comment:
Sorry I'm not familiar with this, why is it needed in this template?

Reply:
Good eyes, @bbirman - It was part of the RN Upgrade Helper's diff, but I rolled it back today during self review since it wasn't required to run the app.